packages/backend/src/rafiki/request/asset.request.ts

```diff
@@ -0,0 +1,88 @@
+import {Asset, AssetEdge, AssetMutationResponse, AssetsConnection} from "../generated/graphql";
```

Comment:
Everything inside `src/rafiki/` is not properly formatted because we ignore the `rafiki` folder.

https://github.com/interledger/testnet/blob/022488fe8426ac448192d320e937fcd94748cdd1/.prettierignore#L12